releasewarrior/templates/fennec/release.json.tmpl

```diff
@@ -24,14 +24,14 @@
             },
             {
                 "alias": "pushapk",
-                "description": "run pushapk",
-                "docs": "https://github.com/mozilla/releasewarrior/blob/master/how-tos/fennec-temp-relpro.md#run-pushapk-manually",
```

Comment:
I think we still need to fix up fennec release. For <=57, we still want these tasks right?

 https://github.com/mozilla/releasewarrior/blob/master/templates/fennec_release.md.tmpl#L29-L30